### PR TITLE
Support for checking bluetooth state on Android

### PR DESCRIPTION
**Pull request by ca057**
Hello Johannes,

I added Android support for the module / package (addresses issue #1). I adjusted the `package.json` and the module structure to load the code only from one file and added it to the example.

On Android there are different bluetooth states than on iOS, I could not match all of them (see README.md). The native code on Android could simply be extended to support all possible states, e.g. turning on/turning off or connecting/disconnecting (see the [`BluetoothAdapter` documentation]https://developer.android.com/reference/android/bluetooth/BluetoothAdapter.html) for further reference).

Please let me know if you want to know more about my implementation or if we should change/improve some parts of the code! 😃

### CODE REVIEW

File: Example/ios/Example/AppDelegate.m [hunk 1]
```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://192.168.179.57:8081/index.ios.bundle?platform=ios&dev=true"];
+  jsCodeLocation = [NSURL URLWithString:@"http://192.168.50.35:8081/index.ios.bundle?platform=ios&dev=true"];
```
Comment (frostney) [inline Example/ios/Example/AppDelegate.m:34]: Nitpick: Could we leave the IP at its original state please? 😄 
At some point I need to update the example to React Native 0.29+ anyway which gets rid of defining the defining the IP in `AppDelegate` anyway.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
   "name": "react-native-bluetooth-state",
-  "version": "2.0.0",
+  "version": "2.1.0",
+  "description": "React Native Module for reading the bluetooth state on iOS and Android (fork of frostney/react-native-bluetooth-state)",
+  "author": "Johannes Stein (http://github.com/frostney)",
+  "contributors": "Christian Ost",
   "repository": {
     "type": "git",
-    "url": "https://github.com/frostney/react-native-bluetooth-state.git"
+    "url": "https://github.com/Artirigo/react-native-bluetooth-state.git"
```
Comment (frostney) [inline package.json:9]: Nitpick: Could we leave the `repository` and `homepage` URL at its original state please? 😄

**Comment by frostney** [comment]
This is amazing work, thank you so much 👍 
This is truly greatly appreciated.

On thing I want to ask: Is the license change absolutely necessary?

There are couple of ideas I have, but don't need to be done in this pull request. I rather have this in merged in first and think about improvements later. The things on my mind are:
- Have a single codebase for the iOS and Android example
- Update the Android example to compatible with React Native 0.29+

Apart from these nitpicks, I would say :shipit: and then see where we're at.

P.S.: Don't worry about the failing Travis build, this repo isn't correctly hooked up to Travis.

File: Example/ios/Example/AppDelegate.m (hunk 1 above)
Comment (ca057) [inline Example/ios/Example/AppDelegate.m:34]: Sure, just changed it for local testing and never set it back 😑 
As updating React Native makes sense, this shouldn´t be an issue for long.

File: package.json (hunk 2 above)
Comment (ca057) [inline package.json:9]: Also here: Sure!
Is seen as a temporary change from me to fit the other repository, would absolutely set it back when closing the pull request and before publishing it to npm!

**Comment by ca057** [comment]
Thanks!
The license change is not necessary and can be set back to the previous version, together with the other changes I made to the package.json. The changes I made to the package.json were mentioned to fit my fork and its repository. (Also the README needs to be adapted 😃 )

**Pull request closed without merging**